charts/consul/test/acceptance/tests/fixtures/cases/static-client-partition/patch.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: static-client
+spec:
+  template:
+    metadata:
+      annotations:
+        "consul.hashicorp.com/connect-inject": "true"
+        "consul.hashicorp.com/connect-service-upstreams": "static-server.default.secondary"
```

Comment:
I guess we can change it later since it's not tested right now, but I think it'll need a port too.